react/features/room-lock/components/PasswordRequiredPrompt.web.js

```diff
@@ -95,17 +103,30 @@ class PasswordRequiredPrompt extends Component<Props, State> {
      * @protected
      */
     _renderBody() {
+        const onIcon = (<Icon
```

Comment:
Could you please only define one icon and set `alt` and `src` dynamically based on `state.passwordShown`? (nit: Also extract the state variable above to avoid repeating the `this.state` too much)

I.e.
```
const { passwordShown } = this.state;
const eyeIcon = (
<Icon
            alt = { passwordShown ? 'Hide Password' : 'Show Password' }
            className = 'textfield-icon'
            onClick = { this._onClickPasswordVisibility }
            src = { passwordShown ? IconAudioOnlyOff : IconAudioOnly } />
);
```